### PR TITLE
Styling

### PR DESCRIPTION
**Pull request by Olamarx**
- [ ] Added styling according to the wireframe
- [ ] Used an external library for the toolbar

### The view
stn

### CODE REVIEW

File: src/components/Nav.module.css [hunk 1]
```diff
@@ -8,8 +8,14 @@
```
Comment (AnselemOdims) [inline src/components/Nav.module.css:10]: - [ ] Please kindly use the same color and font as it used in the original design.

### Your Design
image

### Original Design
image

File: src/components/BookComponent/eachBook.module.css [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #121212;
+}
+
+.title {
+  width: 12.313rem;
```
Comment (AnselemOdims) [inline src/components/BookComponent/eachBook.module.css]: - [ ] Please kindly ensure that your design meets the project design requirements(colors, font, padding, button, spacing etc)

### Your Design
image

### Original Design
image

File: src/components/BookComponent/AddBook.module.css [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+.container form {
+  display: flex;
+  justify-content: space-between;
+}
+
+.container form > input:nth-child(1) {
+  width: 23rem;
+  height: 30px;
+  border-radius: 4px;
```
Comment (AnselemOdims) [inline src/components/BookComponent/AddBook.module.css]: - [ ] Please kindly ensure that your design meets the project design requirements(colors, font, padding, button, spacing etc)

### Your Design
image

### Original Design
image

**Review by AnselemOdims — CHANGES_REQUESTED** [review]
### Hi @Olamarx , 
You have been doing a tremendous job so far :ok_hand:
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
PR has a title and summary. :heavy_check_mark:
No linter errors :heavy_check_mark:

## Required Changes ♻️
- [ ] Please also check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity! You can tag me using @AnselemOdims for a quicker response

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: src/components/BookComponent/eachBook.module.css (hunk 2 above)
Comment (Olamarx) [reply src/components/BookComponent/eachBook.module.css]: @AnselemOdims Thank you, I have fixed this up.

File: src/components/BookComponent/AddBook.module.css (hunk 3 above)
Comment (Olamarx) [reply src/components/BookComponent/AddBook.module.css]: @AnselemOdims Thank you, I have fixed this up but there is a difference in the author name which is not on the wireframe.

File: src/components/Nav.module.css (hunk 1 above)
Comment (Olamarx) [reply src/components/Nav.module.css:10]: @AnselemOdims thank you, I have fixed this up.

**Review by EricMbouwe — APPROVED** [review]
Hi @Olamarx, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Your project looks similar to the wireframe
- Well done making the requested changes
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**